src/coreclr/hosts/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 include_directories(inc)
 
 if(CLR_CMAKE_HOST_WIN32)
-  add_subdirectory(corerun)
   add_subdirectory(coreshim)
 else(CLR_CMAKE_HOST_WIN32)
   add_definitions(-D_FILE_OFFSET_BITS=64)
-  add_subdirectory(unixcorerun)
```

Comment:
https://docs.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting#create-a-host-using-coreclrhosth has a link to this directory. The link should be deleted or updated.

Reply:
Will update when merged.

Reply:
https://github.com/dotnet/docs/pull/23344